### DISCUSSION TITLE
Party compatibility

### DISCUSSION THREAD

**Discussion by liexes** [diamond-rip/Eden#180] [Ideas]
My suggestion is allow compatiblity with another plugin of partys

How party and friends or parties system, etc...

Thx, liexes

**Comment by GoodestEnglish** [comment]
Since I don't own party and friends, plus they use mysql instead of mongodb, this is pretty hard to be added.
I will consider this, but it will be in very low priority

**Reply by liexes** [reply]
It is something quite useful for servers that use the plugin that would already be networks, this is what I say, it would attract more people to the plugin. In addition, the party and friends plugin has a free version.
